From a53690cdcb5c5e453ff7179306309875780e3ff8 Mon Sep 17 00:00:00 2001
From: Bruno Sampaio <bruno.sampaio08@unifesp.br>
Date: Tue, 21 May 2024 21:15:00 -0300
Subject: [PATCH 1/2] make finder-test.sh use $PATH execs for assignment-4.2

---
 finder-app/finder-test.sh | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/finder-app/finder-test.sh b/finder-app/finder-test.sh
index 16b8865..7737ba5 100755
--- a/finder-app/finder-test.sh
+++ b/finder-app/finder-test.sh
@@ -55,10 +55,10 @@ fi
 
 for i in $( seq 1 $NUMFILES)
 do
-	./writer "$WRITEDIR/${username}$i.txt" "$WRITESTR"
+	writer "$WRITEDIR/${username}$i.txt" "$WRITESTR"
 done
 
-OUTPUTSTRING=$(./finder.sh "$WRITEDIR" "$WRITESTR")
+OUTPUTSTRING=$(finder.sh "$WRITEDIR" "$WRITESTR")
 
 # remove temporary directories
 rm -rf /tmp/aeld-data
-- 
2.45.1

