From 1e3732cb7f1dd20964d9a99d58c6745e1a36c5ab Mon Sep 17 00:00:00 2001
From: Bruno Sampaio <bruno.sampaio08@unifesp.br>
Date: Fri, 24 May 2024 16:11:01 -0300
Subject: [PATCH] fix makefile all:

---
 server/Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/server/Makefile b/server/Makefile
index 8bd1a61..79dd73b 100644
--- a/server/Makefile
+++ b/server/Makefile
@@ -7,7 +7,7 @@ aesdsocket:# | server-dir
 #server-dir:
 #	mkdir -p ../server
 
-all: aesdsocker
+all: aesdsocket
 
 clean:
 	rm -f aesdsocket
-- 
2.45.1

